lib/bundlex.ex

```diff
@@ -47,7 +81,7 @@ defmodule Bundlex do
   @doc """
   Returns family of the platform obtained with `platform/0`.
   """
-  @spec family() :: :unix | :windows
+  @spec family() :: :unix | :windows | :custom
```

Comment:
Since we deprecate this function anyway, maybe let's keep it unchanged for better compatibility?

Reply:
we deprecate platform, not family